charts/mokta/values.yaml

```diff
@@ -0,0 +1,21 @@
+image: public.ecr.aws/citizensadvice/mokta:3.0.0.pre1
+
+env:
+  SAFE_HOSTS: https://*.qa.citizensadvice.org.uk
+  OKTA_EMBED_LINK: https://cadev.oktapreview.com/home/oidc_client/0oa1hwcw23bWFpbHf0h8/aln5z7uhkbM6y7bMy0g7
```

Comment:
repo is public, is it safe to leave this here?

Reply:
Hmmm, maybe best not to

Reply:
Although that is the link you get redirected to so it is isn't secret.